gemrb/GUIScripts/iwd/CharGen.py

```diff
@@ -479,6 +474,7 @@ def SetCharacterDescription():
 		MageSpell = CommonTables.ClassSkills.GetValue (ClassName, "MAGESPELL")
 		IsBard = CommonTables.ClassSkills.GetValue (ClassName, "BARDSKILL")
 		IsThief = CommonTables.ClassSkills.GetValue (ClassName, "THIEFSKILL")
+		Alignment = GemRB.GetPlayerStat (MyChar, IE_ALIGNMENT)
```

Comment:
I don't see this used anywhere — looks redundant.

Reply:
Fixed.